.gitignore

```diff
@@ -7,3 +7,4 @@
 node_modules
 oclif.manifest.json
 /output
+/lambda/data
```

Comment:
Rebase against main branch. Already there. Although, I appear to have misspelt the rule for the keep file. oops. !/labda/data/.keep should be !/lambda/data/.keep